Server/API/login/postLogin.go

```diff
@@ -0,0 +1,41 @@
+package login
+
+import (
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/dgrijalva/jwt-go"
+	"github.com/labstack/echo/v4"
+	entities "ishtaloo.io/Entities"
+)
+
+func postLogin(c echo.Context) (err error) {
+	cookie := new(http.Cookie)
+	cookie.Name = "userId"
+	u := new(entities.LoginRequest)
+	if err = c.Bind(u); err != nil {
+		return err
+	}
+	userName := entities.LoginRequest{
+		UserName: u.UserName,
+	}
+	cookie.Value, _ = createToken(userName)
+	cookie.HttpOnly = true
+	c.SetCookie(cookie)
+	return c.String(http.StatusOK, "login successful")
+}
+
+func createToken(userName entities.LoginRequest) (string, error) {
+	var err error
+	atClaims := jwt.MapClaims{}
+	atClaims["authorized"] = true
+	atClaims["user_name"] = userName.UserName
+	atClaims["exp"] = time.Now().Add(time.Hour * 24).Unix()
+	at := jwt.NewWithClaims(jwt.SigningMethodHS256, atClaims)
+	token, err := at.SignedString([]byte(os.Getenv("ACCESS_SECRET")))
```

Comment:
Send me the env

Reply:
Add "ACCESS_SECRET=jdnfksdmfksd" to the server env